appender/elasticsearch-rest/src/main/java/org/apache/karaf/decanter/appender/elasticsearch/rest/ElasticsearchAppender.java

```diff
@@ -139,6 +148,15 @@ public void close() {
     @Override
     public void handleEvent(Event event) {
         try {
+            if (indexFilter != null && !indexFilter.trim().equals("")) {
+                String[] check = indexFilter.trim().split(",");
+                for (String filter : Arrays.asList(check)) {
+                    String[] split = filter.trim().split(":");
+                    if (event.getProperty(split[0]) == null || !event.getProperty(split[0]).equals(split[1])) {
```

Comment:
Maybe use a regex pattern instead of `equals` ? It would give even more control for user.

Reply:
Yes, I was thinking about it but in this case, may be it would be nice to declare the properties like:
```
properties.filter.attribute1=regex value1
properties.filter.attribute2=regex value2
properties.filter.attribute3=regex value3
```
Thoughts ?

Reply:
See my previous comment. I understand the use case now ;)

Reply:
Basically, I would do the following algorithm:

```
for (Property properties : properties) {
  // create include names pattern, exclude names pattern, include values pattern, exclude values pattern
  if (excludeName != null && property.getName().match(excludeName) continue;
  if (excludeValue != null && property.getValue().match(excludeValue) continue;
  if (includeName != null && !properrty.getName().match(includeName) continue;
  if (includeValue != null && !property.getValue().match(includeValue) continue;
  send(event);
}
```